lib/oidcc/plug/client_store.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Oidcc.Plug.ClientStore do
```

Comment:
We need to add some docs explaining how this works.

Reply:
Done